frontend/packages/knative-plugin/src/components/knatify/KnatifyForm.tsx

```diff
@@ -25,14 +24,16 @@ const KnatifyForm: React.FC<FormikProps<FormikValues> & DeployImageFormProps> =
   usePreventDataLossLock(isSubmitting);
 
   return (
-    <Form className="co-deploy-image" data-test-id="knatify-form" onSubmit={handleSubmit}>
-      <ImageSearchSection disabled />
-      <IconSection />
-      <AppSection
-        project={values.project}
-        noProjectsAvailable={projects.loaded && _.isEmpty(projects.data)}
-      />
-      <AdvancedSection values={values} />
+    <FlexForm className="co-deploy-image" data-test-id="knatify-form" onSubmit={handleSubmit}>
+      <FormBody>
```

Comment:
should use flex layout without FlexForm
```suggestion
      <FormBody flexLayout>
```